src/aframe-streetmix-parsers.js

```diff
@@ -295,7 +295,8 @@ function createBusElement (isOutbound, positionX) {
 }
 
 function createDriveLaneElement (variantList, positionX, segmentWidthInMeters, length, animated = 'false') {
-  var totalStreetDuration = (length / 5) * 1000;
+  var speed = 10; // meters per second
```

Comment:
this increases the default speed to 10 m/s ~= 22mph

Reply:
very speedy 🐎